### PR TITLE
feat: add Qwik blog

### PR DESCRIPTION
**Pull request by gioboa**
To limit the number of files and the cognitive weight of a big review, I decided to open a PR for the blog skeleton.
We will iterate on ti later.

### CODE REVIEW

File: packages/docs/src/routes/(blog)/blog/components/newsletter.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export const Newsletter = component$(() => {
+  return (
+    <section class="bg-slate-900 text-white py-16 px-4 sm:px-6 lg:px-8 rounded-xl">
+      <div class="max-w-2xl mx-auto text-center">
+        <h2 class="text-3xl font-bold">Stay Updated</h2>
+        <p class="mt-4 text-slate-300">
+          Get the latest articles and insights delivered straight to your inbox.
+        </p>
+
+        <form class="mt-8 flex flex-col sm:flex-row gap-4 justify-center">
```
Comment (gioboa) [inline packages/docs/src/routes/(blog)/blog/components/newsletter.tsx:13]: We can add the subscription to the Qwik Newsletter here @shairez

File: packages/docs/src/routes/(blog)/blog/components/mdx/article-hero/article-hero.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  }
+
+  return (
+    <>
+      <div class="pt-4">
+        <a
+          class="text-[color:var(--text-color)] flex items-center space-x-1 pl-2"
+          target="_blank"
+          rel="noopener"
+          href="/blog"
```
Comment (Varixo) [inline packages/docs/src/routes/(blog)/blog/components/mdx/article-hero/article-hero.tsx:21]: should it open a new tab every time?

**Review by Varixo — COMMENTED** [review]
Great job!

File: packages/docs/src/routes/(blog)/blog/components/mdx/article-hero/article-hero.tsx (hunk 2 above)
Comment (gioboa) [reply packages/docs/src/routes/(blog)/blog/components/mdx/article-hero/article-hero.tsx:21]: Nice catch, let's remain in the same page

**Review by Varixo — APPROVED** [review]

**Pull request merged**